src/app/public/modules/file-attachment/file-attachment.service.ts

```diff
@@ -6,6 +6,9 @@ import {
   SkyFileItem
 } from './file-item';
 
+/**
+ * @internal
```

Comment:
☝️ Not part of the scope of this PR, but noticed these were improperly being rendered in our docs.